os/board/stm32l4r9ai-disco/src/stm32_exflash.c

```diff
@@ -126,27 +127,18 @@ void stm32_exflash_initialize(void)
 #if defined(CONFIG_FLASH_PARTITION)
     int ret;
     struct mtd_dev_s *mtd;
+    partition_info_t partinfo;
 
     mtd = (FAR struct mtd_dev_s *)mtd_initialize();
-    ret = configure_partitions(mtd, &g_flash_part_data);
+    ret = configure_partitions(mtd, &g_flash_part_data, &partinfo);
```

Comment:
This needs to be changed to `configure_mtd_partitions` please.
Causes problem in the current STM32L4r9ai-disco builds as well.

Reply:
Done. Thanks.